numba_dppy/decorators.py

```diff
@@ -60,9 +60,7 @@ def _kernel_jit(signature, debug, access_types):
         ]
     )
 
-    if restype is not None and restype != types.void:
-        msg = "DPPY kernel must have void return type but got {restype}"
-        raise TypeError(msg.format(restype=restype))
+    assert_no_return(rettype)
```

Comment:
A comment will be useful.

Reply:
If comment needed then function should have better name.